### PR TITLE
[feature/OPT-1029] to dev

### PR DESCRIPTION
**Pull request by smaneroiriusrisk**

### CODE REVIEW

File: slp_drawio/tests/unit/parse/transformers/test_parent_calculator_transformer.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+from slp_drawio.slp_drawio.objects.diagram_objects import Diagram, DiagramComponent, DiagramTrustZone
+from slp_drawio.slp_drawio.parse.tranformers.parent_calculator_transformer import ParentCalculatorTransformer
+
+
+class TestParentCalculatorTransformer:
```
Comment (jmgarcia-iriusrisk) [inline slp_drawio/tests/unit/parse/transformers/test_parent_calculator_transformer.py:5]: This class lacks tests to set the parent for:

1. Component has a TrustZone as parent
2. Trustzone has a Component as a parent
3. Trustzone has another TZ as a parent

File: slp_drawio/slp_drawio/parse/tranformers/parent_calculator_transformer.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    DiagramComponent: ParentType.COMPONENT,
+    DiagramTrustZone: ParentType.TRUST_ZONE
+}
+
+
+def get_parent_type(element) -> ParentType:
+    return PARENT_TYPES[element.__class__]
+
+
+class ParentCalculatorTransformer(Transformer):
```
Comment (jmgarcia-iriusrisk) [inline slp_drawio/slp_drawio/parse/tranformers/parent_calculator_transformer.py:15]: Parent calculator should be used at DrawioParser.build_otm bellow  the line # TODO Implement and call Transformers here

**Review by jmgarcia-iriusrisk — CHANGES_REQUESTED** [review]
@smaneroiriusrisk nice work! Great idea how uniting components as trustzone in the same list works!
I wrote a couple of comments I'd like to discuss before approving this PR.

File: slp_drawio/tests/unit/parse/transformers/test_parent_calculator_transformer.py (hunk 1 above)
Comment (smaneroiriusrisk) [reply slp_drawio/tests/unit/parse/transformers/test_parent_calculator_transformer.py:7]: I think these use cases are out of scope of the MVP, but I have done it anyway. I had to modify the DiagramTrustzone class.

File: slp_drawio/slp_drawio/parse/tranformers/parent_calculator_transformer.py (hunk 2 above)
Comment (smaneroiriusrisk) [reply slp_drawio/slp_drawio/parse/tranformers/parent_calculator_transformer.py:20]: I'm not sure if this is the right task to do that, but also done anyway

**Review by jmgarcia-iriusrisk — APPROVED** [review]

**Review by dantolin-iriusrisk — DISMISSED** [review]

**Review by jmgarcia-iriusrisk — APPROVED** [review]

**Review by dantolin-iriusrisk — APPROVED** [review]

**Pull request merged**